packages/lesswrong/lib/collections/posts/custom_fields.ts

```diff
@@ -512,7 +511,7 @@ addFieldsDict(Posts, {
     ...schemaDefaultValue(false),
     viewableBy: ['members'],
     insertableBy: ['members'],
-    editableBy: [userOwns, 'sunshineRegiment', 'admins'],
+    editableBy: ['members', 'sunshineRegiment', 'admins'],
```

Comment:
A slightly nervewracking change to make, but seems correct; edit access is already enforced at the object level so this field-level restriction was a duplicate check.